### PR TITLE
Added description in i button.

### PR DESCRIPTION
**Pull request by PiyushK79**
Fixes issue #1284 

Added descriptions for all categories.
@rupali-codes Please review it and merge it with the main branch.

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by k-deepak04 — CHANGES_REQUESTED** [review]
@PiyushK79 deployment is failing, @rupali-codes please provide him related message so that he can figure out what's going wrong.

**Comment by rupali-codes** [comment]
@PiyushK79 hope this helps

image

**Comment by PiyushK79** [comment]
Alright, I will look into it.

File: components/TopBar/CategoryDescriptions.ts [hunk 1]
```diff
@@ -4,69 +4,69 @@ type CategoryDescriptions = {
...
+  openSourceTools: 'There are various open source tools available for developers. Some of the most popular ones include Hadoop, VLC Media Player, and GIMP1. Other open source tools include TensorFlow, Eclipse Che, and OpenShift Do.',
   // resources
-  blogs: 'Blogs category description',
-  ebook: 'E-Book category description',
-  hosting: 'Hosting category description',
-  officialdocs: 'Official Docs category description',
-  'project ideas': 'Project category description',
+  blogs: 'The term “blog” is short for “weblog.” Blogs are usually maintained by an individual with regular entries of commentary, descriptions of events, or other material such as graphics or video.',
+  ebook: 'An ebook (short for electronic book) is a book publication made available in digital form, consisting of text, images, or both readable on the flat-panel display of computers or other electronic devices.',
+  hosting: 'Web hosting is an online service that makes your website's content accessible on the internet. When you purchase a hosting plan, you are renting space on a physical server to store all the files and data.',
```
Comment (Anmol-Baranwal) [inline components/TopBar/CategoryDescriptions.ts:48]:
```suggestion
  hosting: 'Web hosting is an online service that makes your websites content accessible on the internet. When you purchase a hosting plan, you are renting space on a physical server to store all the files and data.',
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@PiyushK79 

According to me, this is what causing the issue.

Adjust the parenthesis due to conflicting from the wrapped ones.

This would likely fix the issue.

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Pull request merged**